src/viam/utils.py

```diff
@@ -149,6 +165,79 @@ def _convert(v: ValueTypes) -> Any:
     return struct
 
 
+def flat_tensors_to_ndarrays(flat_tensors: FlatTensors) -> Dict[str, NDArray]:
+    property_name_to_dtype = {
+        "float_tensor": np.float32,
+        "double_tensor": np.float64,
+        "int8_tensor": np.int8,
+        "int16_tensor": np.int16,
+        "int32_tensor": np.int32,
+        "int64_tensor": np.int64,
+        "uint8_tensor": np.uint8,
+        "uint16_tensor": np.uint16,
+        "uint32_tensor": np.uint32,
+        "uint64_tensor": np.uint64,
+        "tensor": np.float32,
```

Comment:
I think we can actually remove this field because it shouldn't be possible for `WhichOneof` to even return `"tensor"`

It was my mistake for including it thanks for pointing it out